doc/source/whatsnew/v2.1.0.rst

```diff
@@ -364,9 +364,11 @@ Datetimelike
 - Bug in :func:`date_range` when ``freq`` was a :class:`DateOffset` with ``nanoseconds`` (:issue:`46877`)
 - Bug in :meth:`Timestamp.date`, :meth:`Timestamp.isocalendar`, :meth:`Timestamp.timetuple`, and :meth:`Timestamp.toordinal` were returning incorrect results for inputs outside those supported by the Python standard library's datetime module (:issue:`53668`)
 - Bug in :meth:`Timestamp.round` with values close to the implementation bounds returning incorrect results instead of raising ``OutOfBoundsDatetime`` (:issue:`51494`)
+- Bug in :meth:`Timestamp.weekday` with days incorrectly calculating before year 1752 (:issue:`53738`)
```

Comment:
"Bug in :meth:`Timestamp.weekday` was returning incorrect results before year 1752 (:issue:`53738`)"